lib/file-finder.js

```diff
@@ -113,20 +114,23 @@ class FileFinder {
                         // Add the file path to the cache of each directory searched.
                         for (let j = 0; j < searched; j++) {
                             cache[dirs[j]].push(filePath);
+                            yield [filePath];
```

Comment:
I updated `findAllInDirectoryAndParents` to return strings on each iteration to stay consistent. I also updated the tests too in tests/lib/file-finder.js

I also took out the thisConfig.rootConfig flag I had. However, I am running into errors where there are instances where an empty object is returned:
<img width="1070" alt="screen shot 2017-05-27 at 10 23 01 pm" src="https://cloud.githubusercontent.com/assets/3211873/26525654/307e96f8-432c-11e7-96ce-5cdee319d506.png">

It's happening in  getLocalConfig in the config.js file

Can you help here?